README.md

```diff
@@ -26,11 +26,13 @@ Clone the repository
 
   - Production: `$ fvm flutter run --flavor production`
 
-- Run the app with the desire app flavor:
+- Run unit testing:
 
-  - Staging: `$ fvm flutter run --flavor staging`
+  - `$ fvm flutter test .`
 
-  - Production: `$ fvm flutter run --flavor production`
+- Run integration testing:
+
+  - `$ fvm flutter drive --driver=test_driver/integration_test.dart --target=integration_test/{test_file}.dart --flavor staging`
 
```

Comment:
maybe add an example below would be good:

```
For example:

$ fvm flutter drive --driver=test_driver/integration_test.dart --target=integration_test/my_home_page_test.dart --flavor staging

Reply:
@sleepylee I just add the example for the integration test part. Please help me review it 🙋